app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,87 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$parchment: #ebe5d5;
+$dark-blue: #112230;
+$light-blue: #68b7e1;
+$ocean-green: #70a58b;
+$dark-ocean_green: #649d81;
+$grey: #3d4b50;
+$sand: #b9a167;
+$crab-red: #e24d34;
+$shrimp-pink: #e49e78;
+$dark-shrimp_pink: #e19268;
+$clam-tan: #d7cab7;
+$dark-clam_tan: #c2ae91;
+$dark-dark-clam-tan: #b49b78;
+$seafood-parchment: #e8e1d4;
+$white: #ffffff;
+
+body {
+    background-color: $parchment;
+}
+
+h1, h2 {
+    color: $ocean-green;
+}
+
+#big-title {
+    font-color: $ocean-green;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Unknown property font-color